docs/how-to/testing-with-minio.md

```diff
@@ -0,0 +1,133 @@
+# Testing with Minio
+
+```{warning}
+The Minio charm is not suited for production usage. If that is what you are after, you should have a look at [Charmed Ceph](https://ubuntu.com/ceph/docs) and then follow [this guide](https://discourse.charmhub.io/t/tempo-ha-docs-how-to-use-ceph-backed-s3-storage-for-ha-charms/15740) instead.
```

Comment:
I'd highlight this is not about the charm per se, but the upstream. We recommend using ceph (perhaps good to hint at why?)

```suggestion
Minio is not suited for production usage. If that is what you are after, you should have a look at [Charmed Ceph](https://ubuntu.com/ceph/docs) and then follow [this guide](https://discourse.charmhub.io/t/tempo-ha-docs-how-to-use-ceph-backed-s3-storage-for-ha-charms/15740) instead.
```

Reply:
Well, it is mainly about the charm.